sdk-extensions/jfr-events/src/main/java/io/opentelemetry/sdk/extension/jfr/ScopeEvent.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.jfr;
+
+import io.opentelemetry.api.trace.SpanContext;
+import jdk.jfr.Category;
+import jdk.jfr.Description;
+import jdk.jfr.Event;
+import jdk.jfr.Label;
+import jdk.jfr.Name;
+
+@Name("io.opentelemetry.context.Scope")
+@Label("Scope")
+@Category("Open Telemetry Tracing")
+@Description(
+    "Open Telemetry trace event corresponding to the span currently "
+        + "in scope/active on this thread.")
+class ScopeEvent extends Event {
+
+  @Label("Trace Id")
```

Comment:
Does `@Label` work on methods? Very small nit, but tend to like putting annotations on public API where possible. Alternatively, if we annotate the fields maybe we don't need the getters.